packages/testing/addon/mirage-graphql/schema.graphql

```diff
@@ -2652,126 +2963,187 @@ type PageInfo {
+    before: String = null
+    after: String = null
+    first: Int = null
+    last: Int = null
+    metaValue: [JSONValueFilterType] = null
+    slug: String = null
+    name: String = null
+    description: String = null
+    type: TaskTypeArgument = null
+    isArchived: Boolean = null
 
     """
     TaskOrdering
     """
-    orderBy: [TaskOrdering]
-    filter: [TaskFilterSetType]
-    order: [TaskOrderSetType]
-    createdByUser: String
-    createdByGroup: String
-    modifiedByUser: String
-    modifiedByGroup: String
+    orderBy: [TaskOrdering] = null
+    filter: [TaskFilterSetType] = null
+    order: [TaskOrderSetType] = null
+    createdByUser: String = null
+    createdByGroup: String = null
+    modifiedByUser: String = null
+    modifiedByGroup: String = null
 
     """
     Only return entries created after the given DateTime (Exclusive)
     """
-    createdBefore: DateTime
+    createdBefore: DateTime = null
 
     """
     Only return entries created at or before the given DateTime (Inclusive)
     """
-    createdAfter: DateTime
-    metaHasKey: String
-    search: String
+    createdAfter: DateTime = null
+    metaHasKey: String = null
+    search: String = null
   ): TaskConnection
   allCases(
-    before: String
-    after: String
-    first: Int
-    last: Int
-    metaValue: [JSONValueFilterType]
-    workflow: ID
+    offset: Int = null
+    before: String = null
+    after: String = null
+    first: Int = null
+    last: Int = null
+    metaValue: [JSONValueFilterType] = null
+    workflow: ID = null
 
     """
     CaseOrdering
     """
-    orderBy: [CaseOrdering]
-    filter: [CaseFilterSetType]
-    order: [CaseOrderSetType]
-    createdByUser: String
-    createdByGroup: String
-    modifiedByUser: String
-    modifiedByGroup: String
+    orderBy: [CaseOrdering] = null
+    filter: [CaseFilterSetType] = null
+    order: [CaseOrderSetType] = null
+    createdByUser: String = null
+    createdByGroup: String = null
+    modifiedByUser: String = null
+    modifiedByGroup: String = null
 
     """
     Only return entries created after the given DateTime (Exclusive)
     """
-    createdBefore: DateTime
+    createdBefore: DateTime = null
 
     """
     Only return entries created at or before the given DateTime (Inclusive)
     """
-    createdAfter: DateTime
-    metaHasKey: String
-    id: ID
-    documentForm: String
-    documentForms: [String]
-    hasAnswer: [HasAnswerFilterType]
-    workItemDocumentHasAnswer: [HasAnswerFilterType]
-    rootCase: ID
-    searchAnswers: [SearchAnswersFilterType]
-    status: [CaseStatusArgument]
+    createdAfter: DateTime = null
+    metaHasKey: String = null
+    id: ID = null
+    documentForm: String = null
+    documentForms: [String] = null
+    hasAnswer: [HasAnswerFilterType] = null
+    workItemDocumentHasAnswer: [HasAnswerFilterType] = null
+    rootCase: ID = null
+    searchAnswers: [SearchAnswersFilterType] = null
+    status: [[CaseStatusArgument]] = null
```

Comment:
Status is an array of arrays?

Reply:
Again, auto-generated. This might be a backend issue